packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -243,8 +243,8 @@ class CreateCommand extends CreateBase {
       macos: featureFlags.isMacOSEnabled && platforms.contains('macos'),
       windows: featureFlags.isWindowsEnabled && platforms.contains('windows'),
       windowsUwp: featureFlags.isWindowsUwpEnabled && platforms.contains('winuwp'),
-      // Enable null-safety for sample code, which is - unlike our regular templates - already migrated.
-      dartSdkVersionBounds: sampleCode != null ? '">=2.12.0-0 <3.0.0"' : '">=2.7.0 <3.0.0"'
+      // Enable null safety everywhere.
+      dartSdkVersionBounds: '">=2.12.0 <3.0.0"'
```

Comment:
This change also opted in the templates for the tests/package/plugins - either those need an opt out string or they need to be updated as well

Reply:
Yes, this was a sloppy PR. Tests are passing locally now.

Reply:
Hmm, I should have said something more specific like the create tests were passing. I'm still getting lots of test failures like:
https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket.appspot.com/8851933560784220304/+/steps/run_module_test_ios/0/stdout

@jonahwilliams I think I misunderstood your comment about tests. Do the template changes I made affects tests such as `module_test_ios` ?

Reply:
`module_test_ios` files were migrated in https://github.com/flutter/flutter/pull/78619/commits/b9f2d5ece8d7e53c67f68131d6506d265d3a11f0, and CI is now passing